src/App.tsx

```diff
@@ -59,11 +32,11 @@ export const App: React.FC = () => {
                   </p>
                 )}
 
-                {author && !loaded && (
+                {author && loading && (
                   <Loader />
                 )}
 
-                {author && loaded && hasError && (
+                {author && !loading && error && (
```

Comment:
consider moving long ternaries to a separate const with consistent name and use the const here
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**